runtime/queries/javascript/indents.scm

```diff
@@ -0,0 +1,22 @@
+[
+ (array)
+ (object)
+ (arguments)
+ (formal_parameters)
+
+ (statement_block)
+ (object_pattern)
+ (class_body)
+ (named_imports)
+
+ (binary_expression)
+ (return_statement)
+ (template_substitution)
+ (export_clause)
+] @indent
+
+[
+ "}"
+ "]"
+ ")"
```

Comment:
```suggestion
  "}"
  "]"
  ")"
```

Sorry this is a bit of a nit. I see the single-space indent in some other indent files as well:

- dart
- go
- llvm
- nix
- scala
- zig

At some point I'd really like to write a formatter for tree-sitter queries to make this all moot 😄

Reply:
A formatter would be pretty nice. I must've missed this while converting all the indent queries, thanks for pointing out :)